packages/manager/modules/pci/src/projects/project/instances/add/translations/Messages_es_ES.json

```diff
@@ -124,5 +124,6 @@
   "pci_projects_project_instances_add_attch_floating_ip_banner_localzone_description3": "Cambiar la localización",
   "pci_projects_project_instances_add_attch_floating_ip_checkbox_label_tooltip": "Opción no disponible durante la configuración de varias instancias. Acceda a «Public IP» para asociar una Floating IP a sus instancias.",
   "pci_project_instances_instance_add_region_savings_plan_info": "Las Locals Zones no son compatibles con los Savings Plans.",
-  "pci_projects_project_instances_add_billing_savings_plan_banner": "Disfrute de tarifas mensuales más bajas gracias a los Savings Plans, conservando la flexibilidad de las instancias por horas."
+  "pci_projects_project_instances_add_billing_savings_plan_banner": "Disfrute de tarifas mensuales más bajas gracias a los Savings Plans, conservando la flexibilidad de las instancias por horas.",
+  "pci_projects_project_instances_add_billing_savings_plan_cta": "Configure sus planes de ahorro"
```

Comment:
This is weird that in some languages it is translated as is, sometimes in half, sometimes entirely